docs/source/conf.py

```diff
@@ -19,10 +19,11 @@
 sys.path.insert(0, os.path.abspath("../.."))
 
 # -- General configuration -----------------------------------------------------
+needs_sphinx = '2.0'
```

Comment:
Please note this is overly restrictive. imgmath is available at least since 1.8.x, probably already earlier (haven't checked in detail). Not a big problem though